File: Famoser.SyncApi/Api/Communication/Response/Base/BaseResponse.cs
using Famoser.SyncApi.Enums;

namespace Famoser.SyncApi.Api.Communication.Response.Base
{
    public class BaseResponse
    {
        public ApiError ApiError { get; set; }
        public string ServerMessage { get; set; }
        public bool RequestFailed { get; set; }
        public bool IsSuccessfull => !RequestFailed;
    }
}

File: Famoser.SyncApi.Webpage/src/Models/Communication/Response/Base/BaseResponse.php
<?php
/**
 * Created by PhpStorm.
 * User: famoser
 * Date: 04.11.2016
 * Time: 19:18
 */

namespace Famoser\SyncApi\Models\Communication\Response\Base;


use Famoser\SyncApi\Types\ApiError;

class BaseResponse
{
    /* @var ApiError $ApiError */
    public $ApiError;

    /* @var string $ServerMessage */
    public $ServerMessage;

    /* @var bool $RequestFailed */
    public $RequestFailed;
}